packages/scss/src/components/inputs/textfield/_textfield.scss

```diff
@@ -223,13 +223,10 @@
 	}
 
 	&.is-required {
-		.textfield-label::after {
-			color: _color("error");
-			content: "*";
-			display: inline-block;
-			font-size: 1.2rem;
-			margin-left: .25em;
-			position: absolute;
+		.textfield-label {
```

Comment:
You should just have .textfield-label;;after as there is no other imbricated style